ci/build_wheel.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+package_name="cucim"
+package_dir="python/cucim"
+package_src_dir="${package_dir}/src/${package_name}"
+
+CMAKE_BUILD_TYPE="release"
```

Comment:
thought: I was going to ask if this was really necessary since it's CMake's default, but now I see that this is for the `build_local` script. It's out of scope for this PR, but maybe that script should support a default since CMake does?

Reply:
There is a default, but it is "debug" rather than release! Could potentially change it if there is no objection from @gigony 
```bash
    local subcommand="${1:-all}"
    local build_type="${2:-debug}"
```

Reply:
Raised as issue: https://github.com/rapidsai/cucim/issues/629